controllers/viewAnswer.js

```diff
@@ -0,0 +1,19 @@
+const models = require("../models/index");
+const viewAnswer = async (req, res) => { 
+
+    try {
+        const question_id = parseInt(req.params.question_id);
+        // const user_id = req.user_id;
+
+        const answers = await models.answers.findAll({ where: { question_id: question_id } });
+
+        return res.status(200).json({ success: 'true', message: answers })
+
```

Comment:
Don't return the answers using message property since its going to bring about insconsistency. We have been using the message property to return strings(either error message or custom messages), so instead do something like
```javascript
 return res.status(200).json({ success: 'true', data: answers })
```